swap/src/asb/config.rs

```diff
@@ -299,18 +292,6 @@ pub fn query_user_for_initial_config(testnet: bool) -> Result<Config> {
     }
     let ask_spread = Decimal::from_f64(ask_spread).context("Unable to parse spread")?;
 
-    let rendezvous_peer_id_str = Input::with_theme(&ColorfulTheme::default())
```

Comment:
Removing this means that the user does not have a chance to configure this during the startup wizard - i.e. the registration with a rendezvous node would be a hidden feature that the user has to activate later by manually filling in config values.
Initially we said that rendezvous should be active by default. I think we should configure default values for it (at least once they are known) and not just omit it.

Reply:
Ok I will add the defaults back